package.json

```
@@ -3,16 +3,15 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www",
+    "start": "nodemon ./bin/www",
```

```suggestion
    "start": "node ./bin/www",
    "dev": "nodemon ./bin/www",
```

dev 환경에서 실행하는 스크립트를 따로 만드는 것이 좋아보입니다.
production 환경에서는 nodemon을 사용하지 않아요.